### PR TITLE
Bump versions of subprojects

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR:
  * Bumps versions of internal Spine SDK dependencies.
  * Improves readability of the documentation for the `scripts/publish-documentation` directory.

### CODE REVIEW

File: scripts/publish-documentation/README.md [hunk 1]
```diff
@@ -136,20 +152,27 @@ After running the example, the following happens:
...
 
 ### OS details
 
 The script was developed under and for the macOS. It should not have problems working on a Linux 
-distribution. However, it was not meant and tested to do so.
\ No newline at end of file
+distribution. However, it was not meant and tested to do so.
+
+[jenv]: https://github.com/jenv/jenv#12-adding-your-java-environment
+[jenv-add]: https://github.com/jenv/jenv#12-adding-your-java-environment
```
Comment (armiol) [inline scripts/publish-documentation/README.md:176]: Isn't this link the same as the previous one? Please review.

**Review by armiol — CHANGES_REQUESTED** [review]
Please see my comment.

File: scripts/publish-documentation/README.md (hunk 1 above)
Comment (tripolkaandrey) [reply scripts/publish-documentation/README.md:176]: It is, indeed. It should be just https://github.com/jenv/jenv.

File: scripts/publish-documentation/README.md [hunk 2]
```diff
@@ -22,9 +34,9 @@ Prerequisites for the target repository:
 - have at least one tag and module.
 
 Prerequisites for running the script:
-- have [`git`](https://git-scm.com/downloads) and [`jenv`](https://github.com/jenv/jenv#12-adding-your-java-environment) installed;
-- have Java **1.8** installed and [registered](https://github.com/jenv/jenv#12-adding-your-java-environment) in `jenv`;
-- have Java **11** installed, registered and [set as global](https://github.com/jenv/jenv#13-setting-a-global-java-version) in `jenv`.
+- have [`git`](https://git-scm.com/downloads) and [`jenv`][jenv-add] installed;
```
Comment (tripolkaandrey) [inline scripts/publish-documentation/README.md:37]: Initially, on line 37, I wanted to have the link to the repository without highlighting any specific part of the documentation. And on line 38, the link to the specific section. That was my intention for having two separate links.

**Comment by tripolkaandrey** [comment]
I have reviewed the changes to the documentation, and aside from my single comment approve them. However, I leave the formal approval to @armiol.

File: scripts/publish-documentation/README.md (hunk 2 above)
Comment (alexander-yevsyukov) [reply scripts/publish-documentation/README.md:37]: I'll add the repo link to come before the "add" thing.

**Comment by alexander-yevsyukov** [comment]
> I have reviewed the changes to the documentation, and aside from my single comment approve them. 
> However, I leave the formal approval to @armiol.

Типа соблюсти субординацию?.. Тут изменений на два чиха? Мы же не хотим все PRs и дальше прогонять через Алекса и меня всё время. Когда-то же надо начинать брать минимальную ответственность за core review.

**Review by tripolkaandrey — APPROVED** [review]
LGTM.

**Review by armiol — APPROVED** [review]

**Pull request merged**